unique_toolkit/unique_toolkit/content/service.py

```diff
@@ -41,32 +49,50 @@ def search_content_chunks(
             search_string (str): The search string.
             search_type (ContentSearchType): The type of search to perform.
             limit (int): The maximum number of results to return.
-            reranker_config (Optional[RerankerConfig]): The reranker configuration. Defaults to None.
             search_language (str): The language for the full-text search. Defaults to "english".
+            reranker_config (Optional[RerankerConfig]): The reranker configuration. Defaults to None.
             scope_ids (Optional[list[str]]): The scope IDs. Defaults to None.
+            chat_only (Optional[bool]): Whether to search only in the current chat. Defaults to None.
 
         Returns:
             list[ContentChunk]: The search results.
         """
-        return self._trigger_search_content_chunks(
-            search_string=search_string,
-            search_type=search_type,
-            limit=limit,
-            reranker_config=reranker_config,
-            search_language=search_language,
-            scope_ids=scope_ids,
-        )
+        if not scope_ids:
+            self.logger.warning("No scope IDs provided for search.")
 
-    @to_async
-    @async_warning
-    def async_search_content_chunks(
+        try:
+            searches = unique_sdk.Search.create(
+                user_id=self.state.user_id,
+                company_id=self.state.company_id,
+                chatId=self.state.chat_id,
+                searchString=search_string,
+                searchType=search_type.name,
+                scopeIds=scope_ids,
+                limit=limit,
+                reranker=reranker_config,  # type: ignore
+                language=search_language,  # type: ignore
```

Comment:
You should now be able to remove the type ignore statements with unique_sdk >=0.9.3.